authentik/stages/email/migrations/0005_alter_emailstage_token_expiry.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.0.12 on 2025-02-27 04:32
+
+import authentik.lib.utils.time
+from authentik.lib.utils.time import timedelta_from_string
+from django.db import migrations, models
+
+
+def convert_integer_to_string_format(apps, schema_editor):
+    EmailStage = apps.get_model("authentik_stages_email", "EmailStage")
+    for stage in EmailStage.objects.all():
+        stage.token_expiry = f"minutes={stage.token_expiry}"
+        stage.save()
```

Comment:
```suggestion
    db_alias = schema_editor.connection.alias
    EmailStage = apps.get_model("authentik_stages_email", "EmailStage")
    for stage in EmailStage.objects.using(db_alias).all():
        stage.token_expiry = f"minutes={stage.token_expiry}"
        stage.save(using=db_alias)
```

Reply:
Also, would it be possible to do something like `EmailStage.objects.using(db_alias).update(token_expiry="minutes="+F("token_expiry"))`? (It might need a Cast in there somewhere)